state-chain/pallets/cf-ingress-egress/src/weights.rs

```diff
@@ -2,25 +2,25 @@
 //! Autogenerated weights for pallet_cf_ingress_egress
 //!
 //! THIS FILE WAS AUTO-GENERATED USING THE SUBSTRATE BENCHMARK CLI VERSION 4.0.0-dev
-//! DATE: 2023-11-03, STEPS: `20`, REPEAT: `10`, LOW RANGE: `[]`, HIGH RANGE: `[]`
+//! DATE: 2024-02-09, STEPS: `20`, REPEAT: `20`, LOW RANGE: `[]`, HIGH RANGE: `[]`
 //! WORST CASE MAP SIZE: `1000000`
-//! HOSTNAME: `ip-172-31-9-222`, CPU: `Intel(R) Xeon(R) Platinum 8275CL CPU @ 3.00GHz`
```

Comment:
Do the benchmarks get ran by the CI at some point and pushed? Am I supposed to leave this file in the PR?